pdns/dnsdistdist/docs/guides/console.rst

```diff
@@ -11,7 +11,7 @@ The console can be enabled with :func:`controlSocket`:
 
   controlSocket('192.0.2.53:5199')
 
-Exposing the console to the network without encryption enabled is not recommended. To enable encryption, first generate a key with :func:`makeKey`::
+Enabling the console without encryption enabled is not recommended. To enable encryption, first generate a key with :func:`makeKey`::
```

Comment:
Maybe adding a word about the need of libsodium for encryption to actually happen would be nice ?

Reply:
We do so a few lines below but you are right in that it would be better to do it here, I'll move it up. Thanks!